dashboard/test/test_helper.rb

```diff
@@ -79,6 +79,7 @@ class ActiveSupport::TestCase
 
   teardown do
     Dashboard::Application.config.action_controller.perform_caching = false
+    I18n.locale = I18n.default_locale
```

Comment:
I picked teardown just because it's specifically undoing something that had (presumably) been done by a test.